generate_rospkg_apkbuild/genapkbuild.py

```diff
@@ -230,9 +230,19 @@ def package_to_apkbuild(ros_distro, package_name,
 
     pkg.evaluate_conditions(os.environ)
 
+    ros2_distros = [
+        name for name, value in get_index(get_index_url()).distributions.items()
+        if value.get("distribution_type") == "ros2"]
+    is_ros2 = ros_distro in ros2_distros
+
     depends = []
     for dep in pkg.exec_depends:
         depends.append(ros_dependency_to_name_ver(dep))
+    if is_ros2:
+        # https://github.com/ros-infrastructure/bloom/blob/1086547b02af00e1f19a244c7fdd6b0e0ec60d20/bloom/generators/rosdebian.py#L99
```

Comment:
Please add a short explanation as a comment.

Reply:
Done in https://github.com/alpine-ros/ros-abuild-docker/pull/158/commits/73a81d913bc048f24a7dc69eeca2cac7aa8c46cc